.mypy.ini

```diff
@@ -5,7 +5,7 @@ error_summary = true
 files =
   packaging/,
   tests/,
-  propcache/
+  src/propcache/
```

Comment:
@bdraco I concluded that it's possible to just have `src/` here and then, this config can be synchronized across projects w/o needing to adjust project-specific names. Also, this list was supposed to be sorted so it'd be `packaging, src, tests`.